.github/workflows/backport-command.yml

```diff
@@ -70,11 +116,178 @@ jobs:
           ORIG_ASSIGNEES: ${{ steps.assignees.outputs.assignees }}
           ORIG_ISSUE_URL: ${{ github.event.client_payload.github.payload.issue.html_url }}
         run: |
-          gh issue create --title "[${BACKPORT_BRANCH}] ${ORIG_TITLE}" \
+          backport_issue_url=$(gh issue list --repo "${TARGET_ORG}/${TARGET_REPO}" \
```

Comment:
should error out if running `/backport` command on an issue that is already a backport. can detect if already a backport issue if the issue title matches regex `\[v[0-9]{2}\.[0-9]{1,2}\.x\]`

Reply:
done